src/components/signupform/signupform.module.css

```diff
@@ -0,0 +1,52 @@
+/* buttons */
+
+.btn {
+  cursor: pointer;
+  color: #fff;
+  background: #645cff;
+  border: transparent;
+  border-radius: 0.25rem;
+  letter-spacing: 1px;
+  padding: 0.375rem 0.75rem;
+  box-shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.1), 0 1px 2px 0 rgba(0, 0, 0, 0.06);
+  transition: 0.3s ease-in-out all;
+  text-transform: capitalize;
+  display: inline-block;
+  width: 100%;
+}
+
+/* form */
+
+.form {
+  width: 90vw;
+  max-width: 600px;
+  background: #fff;
+  border-radius: 0.25rem;
+  box-shadow: 0 4px 6px -1px rgba(0, 0, 0, 0.1),
+    0 2px 4px -1px rgba(0, 0, 0, 0.06);
+  padding: 2rem 2.5rem;
+  margin: 3rem auto;
+}
+.form-label {
```

Comment:
Same point about CSS here @TaylorDJones11 - please return to `camelCase` thanks!

Reply:
> Same point about CSS here @TaylorDJones11 - please return to `camelCase` thanks!

All CSS issues resolved. Far the regex generator, did you use it on the last mentor corner? I came in a bit late. If so I'll check it out from the recording.